index.html

```diff
@@ -0,0 +1,174 @@
+              </li>
+              <li class="nav-item p-2">
+                <a class="nav-a join-us" href="#">Join Us</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+      <section id="main-header">
+        <h3 class="main-header-h3">"Hello! Sharing world"</h3>
+        <h1 class="main-header-h1">ProCoder Full-Stack Diploma 2021</h1>
+        <p class="main-header-p1">
+          Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+          Necessitatibus quos ullam labore at placeat eligendi dolore asperiores
+          in maiores!
+        </p>
+        <h2 class="main-header-h2">2021.12.15 To 2022.6.15</h2>
+        <p class="main-header-p2">
+          @ National Museum of Korea, Art Center Nabi and more
+        </p>
+      </section>
+    </header>
+
+    <main>
+      <section id="main-program">
+        <h2 class="program-h2">Main Program</h2>
+        <ul class="list-unstyled main-program-ul">
+          <li class="program-card">
+            <i class="fas fa-chalkboard-teacher fa-2x icon"></i>
+            <h4 class="program-card-h4">Lecture</h4>
+            <p class="program-card-desc">
+              Learn from the artists from various countries about the art of
+              drawing.
+            </p>
+          </li>
+          <li class="program-card">
+            <i class="fas fa-tv fa-2x icon"></i>
+            <h4 class="program-card-h4">Exhibition</h4>
+            <p class="program-card-desc">
+              Learn from the artists from various countries about the art of
+              drawing.
+            </p>
+          </li>
+          <li class="program-card">
+            <i class="far fa-comments fa-2x icon"></i>
+            <h4 class="program-card-h4">Forum</h4>
+            <p class="program-card-desc">
+              Learn from the artists from various countries about the art of
+              drawing.
+            </p>
+          </li>
+          <li class="program-card">
+            <i class="far fa-edit fa-2x icon"></i>
+            <h4 class="program-card-h4">Workshop</h4>
+            <p class="program-card-desc">
+              Learn from the artists from various countries about the art of
+              drawing.
+            </p>
+          </li>
+          <li class="program-card">
+            <i class="fas fa-network-wired fa-2x icon"></i>
+            <h4 class="program-card-h4">Ignite</h4>
+            <p class="program-card-desc">
+              Learn from the artists from various countries about the art of
+              drawing.
+            </p>
+          </li>
+        </ul>
+        <div class="t">
+          <button type="button" class="program-card-btn">Join Our Program Now</button>
+        </div>
+      </section>
+
+      <section id="featured-instractors">
+        <h2 class="instractors-h2">Featured Instractors</h2>
+        <div class="instractors row">
+          
+        </div>
+      </section>
```

Comment:
- [ ] The section "Featured speakers" is missing from the live version with 2 console errors, kindly check your live demo.

Reply:
Done